src/otio_maya_sequencer_adapter/maya_sequencer.py

```diff
@@ -0,0 +1,111 @@
+# SPDX-License-Identifier: Apache-2.0
+# Copyright Contributors to the OpenTimelineIO project
+
+"""Maya Sequencer Adapter Harness"""
+
+import os
+import subprocess
+
+from .. import adapters
```

Comment:
I suspect this is an outdated statement.
Should perhaps be:
```
import opentimelineio as otio
from otio import adapters
```
or just:
`from opentimelineio import adapters`